tests/integration_tests/test_chat_completion.py

```diff
@@ -405,11 +435,12 @@ def dial_recall_expected(r: ChatCompletionResult):
             query = f"What's the temperature in {' and in '.join(city_names)} in celsius?"
 
             init_messages = [
-                sys("act as a helpful assistant"),
                 user("2+3=?"),
                 ai("5"),
                 user(query),
             ]
+            if not is_llama3(deployment):
```

Comment:
LLama 3 models working consistently with tools, if system prompt is passed

I've tried something like `act as a helpful assistant with tool support`, and it worked time-to-time, but not consistently somehow

I think Bedrock on their side somehow implement some system messages for tools, and we can overlap them

Reply:
I think you need to configure system prompt for tool calling for Llama model yourself.
There is this note in Bedrock doc: https://docs.aws.amazon.com/bedrock/latest/userguide/tool-use-inference-call.html#tool-use-send-tool-info

> Meta has specific recommendations for creating prompts that use tools with Llama 3.1 (or later) models. For more information, see [JSON based tool calling](https://llama.meta.com/docs/model-cards-and-prompt-formats/llama3_1/#json-based-tool-calling) in the Meta documentation.

So `act as a helpful assistant with tool support` wouldn't be enough to enable tool calling.